private-templates-service/client/src/components/AdaptiveCard/AdaptiveCard.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import * as AdaptiveCards from "adaptivecards";
+import { Card, Container, TemplateName } from "../AdaptiveCard/styled";
+
+function getCard() {
+	// Hard coded, will remove and connect to backend in future PR
+	var card = {
```

Comment:
Just [FYI](https://stackoverflow.com/questions/35572895/what-is-the-difference-between-var-and-let-in-typescript).